experiments/extract_all_data.sh

```diff
@@ -0,0 +1,72 @@
+     "text/2019-01-25T03:09:45.988462/text_aic=1.0_arity=3_aux=True_BO=True_CO=True_ES=1_ET=720_HR=0.5_it=20_MF=5_pc=30.0_RS=5000_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=False_rec=True_vec=True_graph=True.pickle"
+     "text/2019-01-26T01:23:35.287813/text_aic=1.0_arity=3_aux=True_BO=True_CO=True_ES=1_ET=720_HR=0.5_it=20_MF=5_pc=30.0_RS=5000_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=False_rec=True_vec=True_graph=True.pickle"
+     "regex/2019-03-04T19:13:10.698186/regex_aic=1.0_arity=3_BO=True_CO=False_ES=1_ET=720_HR=0.5_it=23_MF=10_pc=30.0_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=True_graph=True.pickle"
+     "regex/2019-03-04T19:17:09.170192/regex_aic=1.0_arity=3_BO=True_CO=False_ES=1_ET=720_HR=0.5_it=23_MF=10_pc=30.0_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=True_graph=True.pickle"
+     "regex/2019-03-04T19:21:20.153727/regex_aic=1.0_arity=3_BO=True_CO=False_ES=1_ET=720_HR=0.5_it=23_MF=10_pc=30.0_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=True_graph=True.pickle"
+     "regex/2019-03-04T19:25:56.137631/regex_aic=1.0_arity=3_BO=True_CO=False_ES=1_ET=720_HR=0.5_it=23_MF=10_pc=30.0_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=True_graph=True.pickle"
+     "regex/2019-03-04T19:30:01.252339/regex_aic=1.0_arity=3_BO=True_CO=False_ES=1_ET=720_HR=0.5_it=23_MF=10_pc=30.0_RT=3600_RR=False_RW=False_STM=True_L=1.5_batch=10_TRR=randomShuffle_K=2_topkNotMAP=True_graph=True.pickle"
+     )
+
+for LOG_FILE in ${LOG_FILES[@]} ; do
+    echo "Started extracting data from $LOG_FILE"
+    DOMAIN=$(echo "$LOG_FILE" | awk -F "/" '{print $1}')
+    RUN=$(echo "$LOG_FILE" | awk -F "/" '{print $2}')
+    echo "Domain=$DOMAIN, Run=$RUN"
+    echo "Logging extraction process to $DOMAIN/$RUN/.log"
+    mkdir -p "$DOMAIN/$RUN"  # this is just making a folder called e.g. regex/2019...252339/
+    echo "$LOG_FILE" > "$DOMAIN/$RUN/.log"
+    python3 "$ARTIFACT_BIN_PATH/data_extractor.py" "$EXP_OUTS_PATH/$LOG_FILE" "$DOMAIN/$RUN" >> "$DOMAIN/$RUN/.log" 2>&1
+    echo "Finished extraction from $LOG_FILE"
+done
+
+
+# for the recursive functional programming domain, the domain name is listed as "list" and K.E.
+# found pickles for each of the 10 iterations. Saving them each separately and setting the domain name
+# to "rec-fp"
+# TODO(theoxo): Is this the most sane behaviour?
+for ITERATION in {1..10} ; do
+    LOG_FILE="list/2019-07-11T19:49:10.899159/list_aic=1.0_arity=4_ET=57600_it={$ITERATION}_MF=5_noConsolidation=False_pc=30.0_RW=False_solver=ocaml_STM=True_L=1.0_TRR=unsolved_K=5_topkNotMAP=False_rec=False.pickle"
```

Comment:
this is causing a crash because literal curly braces are showing up in the output.

it should be "...it="$ITERATION"_MF=5..." instead of the curlybraces

Reply:
or maybe throw even more quotes in and do "...""$ITERATION""..." if you really want it to be quoted ig

Reply:
seems like it extracts successfully when you fix this! 🎉

Reply:
Oops lol, fixed in [452aeb8](https://github.com/mlb2251/stitch/pull/87/commits/452aeb8b281b77dd9458ab5519f2e41e15c5e079) (and rec-fp data added in [8d01421](https://github.com/mlb2251/stitch/pull/87/commits/8d014211315a147353c291495e133bbb1dfdd7ad))